stable/cockroachdb/values.yaml

```diff
@@ -51,6 +51,24 @@ PodManagementPolicy: Parallel
 UpdateStrategy:
   type: RollingUpdate
 NodeSelector: {}
+## The cockroachdb chart already provides it's own PodAntiAffinity
+## to make sure no cdb pods run on the same node.  However,
```

Comment:
Either use crdb or (preferably) cockroachdb. cdb is never really used as an abbreviation for cockroachdb